### ISSUE TITLE
VLC install path treated as unicode

### ISSUE THREAD

**Issue by dylhack** [Pigpog/vlc-discord-rpc#26]
## Details
OS: Windows 10 (64-bit)
Node: 12.4.0
NPM: 6.9.0
vlc-discord-rpc: [2.2.0](https://github.com/Pigpog/vlc-discord-rpc/releases/tag/2.2.0)

## Notes
System language: French
```
------------------------------------
An error occurred when launching VLC
------------------------------------
Error: spawn C:Program FilesVideoLANVLClc.exe ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:264:19)
    at onErrorNT (internal/child_process.js:456:16)
    at processTicksAndRejections (internal/process/task_queues.js:84:9) {
  errno: 'ENOENT',
  code: 'ENOENT',
  syscall: 'spawn C:Program FilesVideoLANVLC\u000blc.exe',
  path: 'C:Program FilesVideoLANVLC\u000blc.exe',
  spawnargs: [
    '--extraintf',
    'http',
    '--http-host',
    'localhost',
    '--http-password',
    'khp0t539',
    '--http-port',
    9090
  ]
}
```

**Comment by dylhack** [comment]
Here is a temporary resolution for this issue. Replace config.js in the config folder with [this code.](https://raw.githubusercontent.com/Pigpog/vlc-discord-rpc/a02ae5ec4369e8b502877e38ecf7fa43e5e02636/config/config.js)

**Comment by dylhack** [comment]
Will be fixed in 2.3.
